app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError, OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor} should "
+                                     f"vaccinate before visit cafe")

```

Comment:
The error message for `NotVaccinatedError` could be more descriptive. Consider including the visitor's name in the message to make it clearer who the message is referring to, as suggested by the checklist: 'Use descriptive error messages'.